README.md

```diff
@@ -4,14 +4,16 @@
 [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/spectral?color=blue)](https://www.npmjs.com/package/@stoplight/spectral)
 [![Treeware (Trees)](https://img.shields.io/treeware/trees/stoplightio/spectral)](https://plant.treeware.earth/stoplightio/spectral)
 
-A flexible JSON/YAML linter, with out of the box support for OpenAPI v2 and v3.
+A flexible JSON/YAML linter, with out of the box support for OpenAPI v2/v3 and AsyncAPI v2.
```

Comment:
We gotta finally migrate all these oas/aas specs away to a separate repo. :sweat_smile:

Reply:
Yep this was always going to happen eventually and I think v6.0 would be perfect.

1. Documentation on how to include a ruleset via NPM
2. In v6.0 we split these out into their own modules which are then automatically included, in v6.1 we emit warnings suggesting folks should include them directly
3. Spectral v7.0 they're no longer included by default

Dunno if you could sneak number 1 in between your million other tasks? Take a stab at getting all the information in dont worry about wording, I'll polish it up.

Reply:
> Dunno if you could sneak number 1 in between your million other tasks? Take a stab at getting all the information in dont worry about wording, I'll polish it up.

Yeah, I've got a sample repo with an example ruleset up and running, so can simply copy & paste it.
Will do it tomorrow.